nntrainer/compiler/ini_interpreter.cpp

```diff
@@ -207,8 +207,8 @@ getMergeableGraph(std::shared_ptr<const GraphRepresentation> graph,
   // bool preload =
   //   iniparser_getboolean(ini, (sec_name + ":Preload").c_str(), true);
 
-  bool trainable =
-    iniparser_getboolean(ini, (sec_name + ":Trainable").c_str(), false);
+  const std::string &trainable =
+    iniparser_getstring(ini, (sec_name + ":Trainable").c_str(), "false");
```

Comment:
Not related to the PR but, is the default value false?

Reply:
I am not sure about this either. This is our design choice.
False: this avoids user from creating extra trainable layers (i added some point, didn't want to leave it empty).
True: as this is training framework, this just sounds sensible. This also matches with each layer using `trainable=true` by default.

Reply:
I think true as default is better. what do you think?

Reply:
I agree :) let's change this to true

Reply:
Changed to true as a new commit in this PR.

Reply:
True. I guess.